### ISSUE TITLE
Post Selection Pick-List Shrinkage 

### ISSUE THREAD

**Issue by cmrn-rhi** [Public-Health-Bioinformatics/covid19ValidationGrid#25]
Once something is selected from a pick-list, if you try to change it (say you clicked the wrong one by accident) the drop-down menu only shows your current selection, and any existing subclasses, as options. Re-selecting doesn't remove the current selection either.

If this cannot be changed it should be noted somewhere in the SOP (as much as it may seem obvious to some that the user should just delete it - causing the pick-list to reform - this may not occur to all users).

**Comment by ddooley** [comment]
One possibility for this, though an extra click usability wise, is to use the same control that "multi" select lists have.  (There could be two side effect benefits here - the popup selection list will in the future also: A) accommodate "other" option for free-text, b) also use synonym searching (e.g. type "chicken" and get "galus galus")

**Comment by wwhsiao** [comment]
I think if you delete the entry then the pick-list refreshes no? Not ideal either but I think we should minimize needing to have extra clicks by default. Maybe extra menu can be trigger by left click?

**Comment by ivansg44** [comment]
Yes, it does refresh when you delete it. I think ideally, the dropdown would list all items as soon as you start editing a cell, regardless of what value is in it, and then only begin to filter when you start typing. It currently filters immediately.

I don't think this is a pressing enough issue to bring in the pop-up chosen editor. In the future, when we begin looking at custom editors (not as easy as it sounds--I tried it, but probably inevitable if we ever try to get rid of the pop-up editor for multi-select values), we can revisit this. Will probably need to extend the autocomplete editor, and introduce a tweak somewhere.

**Comment by ddooley** [comment]
This is fixed now in dropdowns.
